docs/decision-maker-transaction.md

```diff
@@ -1,6 +1,6 @@
 This guide can be considered as a part 2 of the <a href="../standalone-transaction/">the stand-alone transaction demo</a>. The main difference is that now we are going to use the decision-maker to sign the transaction.
 
-First, import the libraries and the set the constant values. (Get the packages directory from the AEA repository `svn export https://github.com/valory-xyz/open-aea.git/trunk/packages`.)
+First, import the libraries and the set the constant values. (Get the packages directory from the AEA repository `aea fetch PUBLIC_ID_OR_HASH`.)
```

Comment:
also doesn't make sense. You can see below it only uses one protocol. The `aea add ..` command could be used. But then it's in an agent project. Need to find another way.